dora/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -145,6 +150,21 @@ public static LocalCacheManager create(CacheManagerOptions options,
     mInitService =
         options.isAsyncRestoreEnabled() ? Optional.of(Executors.newSingleThreadExecutor()) :
             Optional.empty();
+    if (options.isTtlEnabled()) {
+      mTtlEnforcerExecutor = Optional.of(newScheduledThreadPool(1));
```

Comment:
We have TTL based cache eviction of metadata as well in Dora. Independent eviction of metadata and data can cause conflicts, so we need a way to unify the eviction of metadata and data. cc @ChunxuTang @huanghua78